src/__tests__/utils.test.ts

```diff
@@ -40,3 +42,137 @@ describe("fromWei()", () => {
     expect(fromWei(oneETH, 20).toFixed()).to.be.equal("0.01");
   });
 });
+
+describe("transerToSummary", () => {
+  it("works for integer native currency", () => {
+    const transfers: Payment[] = [
+      {
+        tokenAddress: null,
+        amount: new BigNumber(1),
+        receiver: testData.addresses.receiver1,
+      },
+      {
+        tokenAddress: null,
+        amount: new BigNumber(2),
+        receiver: testData.addresses.receiver2,
+      },
+      {
+        tokenAddress: null,
+        amount: new BigNumber(3),
+        receiver: testData.addresses.receiver3,
+      },
+    ];
+    const summary = transfersToSummary(transfers);
+    // For some reason expect.to.equal() doesnt work for BigNumbers.
```

Comment:
PS - this comment appears to be pasted several times in each test here. Once the one is fixed, I would think the comment can be removed everywhere.

Reply:
The problem for expect(amount.eq(new BigNumber(6))).to.be.true or something like that is that the output if it fails is ugly. I think it will just say "Expected true, got false".
Without debugging we cannot find out what the unexpected value was.

The string comparison workaround now will at least show "Expected 6, got 4".

Reply:
I see what you mean here. Its cool if you wanna leave it, but maybe we could take out the pasted comments everywhere.